site/assets/scss/_component-examples.scss

```diff
@@ -114,11 +115,12 @@
 //
 
 .bd-example-row [class^="col"],
-.bd-example-cssgrid .grid > * {
+.bd-example-cols [class^="col"] > *,
+.bd-example-cssgrid [class*="grid"] > * {
   padding-top: .75rem;
   padding-bottom: .75rem;
-  background-color: rgba(var(--bd-violet-rgb), .1);
-  border: map-get($border-widths, 1) solid rgba(var(--bd-violet-rgb), .25);
+  background-color: rgba(var(--bd-violet-rgb), .15);
+  border: map-get($border-widths, 1) solid rgba(var(--bd-violet-rgb), .3); // Boosted mod: use of `map-get($border-widths; 1)` instead of `1px`
```

Comment:
Should we reintroduce `1px` here ?

Reply:
In all this file I'm wondering if we shouldn't replace `map-get($border-widths; 1)` by `$border-width` (or by the corresponding CSS var if it exists) since in the end we probably want to have the same border width as defined in Boosted. If it changes at some point from 1px to 2px, it'll be automatically changed in the docs here as well. What do you think?

Reply:
Finally let's go with `1px` + modify this file to drop any `map-get($border-widths; 1)` occurrences.